manifests/agent/hiera.pp

```diff
@@ -24,8 +24,14 @@
       }
     }
   }
-
   $hieradata = "${codedir}/hieradata"
+  
+  # ensure we've got a target to point our shortcuts to
+  dirtree { "${workdir}/hieradata":
```

Comment:
Do ownership and mode need to be managed on the directory tree as well?

Reply:
at this point no, this is sufficient. (I get nervous managing things that *might be* managed by the PE module with `file` resources because we've been burned in the past when they suddenly started managing something that they weren't before.)

Longer term, this whole thing needs redesigned anyway, this is just keeping it functional for today.